pymc3/model.py

```diff
@@ -434,6 +439,23 @@ def __init__(self, name='', model=None):
             self.potentials = treelist()
             self.missing_values = treelist()
 
+        if theano_config is None:
+            theano_config = {'compute_test_value': 'raise'}
+        self._theano_config = theano.configparser.change_flags(**theano_config)
+        # We need to enter the theano config, since variables could be
+        # created in the init of subclasses.
+        self._theano_config.__enter__()
```

Comment:
Yes, I think I do. The problem is that subclasses of `Model` are allowed to add variables in `__init__`, so `__enter__` is called before `Model.__init__` using some metaclass magic. But maybe it would be better to move this to the metaclass instead of doing it in init.